### PR TITLE
Edit title fix for Notebook and Paragraph

### PR DESCRIPTION
**Pull request by r-kamath**
- enable autofocus to input field *(no need to click twice to edit title)*
- fix esc key handler on edit input to revert the change *(edit and hit Esc to reset the title back to the old one)*
- save title on enter and blur

Before:
before
After:
after

### CODE REVIEW

File: zeppelin-web/src/app/notebook/notebook.css [hunk 1]
```diff
@@ -84,12 +84,10 @@
   color: #555;
   background: #fff;
   border: 1px solid #ccc;
-  border-radius: 4px;
+  border-radius: 0;
   -webkit-box-shadow: inset 0 1px 1px rgba(0, 0, 0, .075);
   box-shadow: inset 0 1px 1px rgba(0, 0, 0, .075);
-  -webkit-transition: border-color ease-in-out .15s, -webkit-box-shadow ease-in-out .15s;
```
Comment (r-kamath) [inline zeppelin-web/src/app/notebook/notebook.css]: @corneadoug @prabhjyotsingh @Leemoonsoo Removed transition for both note and paragraph title, was causing the flicker while updating the title and the animation wasn't visible (Ref: Before screenshot)

**Comment by prabhjyotsingh** [comment]
@r-kamath since we are doing this, can you add animation to the input box like its here http://www.w3schools.com/bootstrap/bootstrap_forms_inputs.asp

**Comment by r-kamath** [comment]
@prabhjyotsingh adding transition requires too many other changes. Planning to do it in a different ticket/PR

**Comment by corneadoug** [comment]
I just checked that PR, a few comments:
* When we have a new paragraph title **untitled**, the value is not inside the input on edit.
* When the inputs appear, it makes the element height and vertical position change

**Comment by r-kamath** [comment]
@corneadoug fixed

**Comment by corneadoug** [comment]
And I still have the element height issue for the Notebook Title

**Comment by corneadoug** [comment]
Can you also update the license file?
https://github.com/apache/incubator-zeppelin/blob/b5e2e62f239d6b1b625add16405e3a302e8ff060/zeppelin-distribution/src/bin_license/LICENSE

**Comment by corneadoug** [comment]
CI seems to be failing for the same reason as my PR #324

**Comment by r-kamath** [comment]
@corneadoug try merging master on #324

**Comment by corneadoug** [comment]
@r-kamath Could you rebase? Seems there is some conflicts

**Comment by corneadoug** [comment]
Merging if there is no more discussions

**Pull request closed without merging**